### ISSUE TITLE
[Feature] Add audio fields to audio search response

### ISSUE THREAD

**Issue by krysal** [WordPress/openverse-api#212]
## Problem

Currently the audio search endpoint (`http://localhost:8000/v1/audio?q=<search_term>`) returns an audio list with the same generic data for images + audio links but we're missing the `duration` and `category`, required to show the track component with complete info in the [audio search list](https://wordpress.org/openverse/search/audio). 

```json
{
      "id": "02ea0f38-5426-452d-8b9a-49fa00a05df6",
      "title": "I Love U",
      "foreign_landing_url": "https://www.jamendo.com/track/14428",
      "creator": "Dance2003",
      "creator_url": "https://www.jamendo.com/artist/1799/dance2003",
      "url": "https://mp3d.jamendo.com/download/track/14428/mp32",
      "license": "by-nc-sa",
      "license_version": "2.5",
      "license_url": "https://creativecommons.org/licenses/by-nc-sa/2.5/",
      "provider": "jamendo",
      "source": "jamendo",
      "tags": [
        {
          "name": "instrumental"
        }
      ],
      "fields_matched": [
        "title"
      ],
      "thumbnail": "http://localhost:8000/v1/audio/02ea0f38-5426-452d-8b9a-49fa00a05df6/thumb/",
      "waveform": "http://localhost:8000/v1/audio/02ea0f38-5426-452d-8b9a-49fa00a05df6/waveform/",
      "detail_url": "http://localhost:8000/v1/audio/02ea0f38-5426-452d-8b9a-49fa00a05df6/",
      "related_url": "http://localhost:8000/v1/audio/02ea0f38-5426-452d-8b9a-49fa00a05df6/related/"
}
```

## Description

Add the `duration` and `category` to each audio object.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by sarayourfriend** [comment]
@krysal just to make sure I'm understanding this ticket right, is it still true that the audio doesn't return the duration? It looks like the `AudioSerializer` has a `duration` field, am I looking in the wrong spot?

**Comment by sarayourfriend** [comment]
Okay! Thanks for the instructions. I'll start taking a look 🙂

**Comment by krysal** [comment]
You're right. Actually, I just mentioned `duration` and `category` because is what we're needing right now for the audio results page, these fields are part of the AudioTrack component, but we can also add all the audio related fields at once.

Thanks for taking a look at this!

**Comment by obulat** [comment]
The ElasticSearch mapping does not contain all the fields. I think we've only added the ones that are relevant for searching. However, this means that ES returns only the mapped fields, and if we want other fields as well (eg `duration`), we would need to query the database on request.

There probably are two solutions:
1. Add all audio fields to the ES mapping so that all fields are returned in ES search results.
2. After receiving ES search results, call the DB to get the missing fields. 
I wonder which would be better for performance? My guess is the first option, but I am not sure.
